plugins/ltac/g_rewrite.mlg

```diff
@@ -20,6 +20,7 @@ open Genintern
 open Geninterp
 open Extraargs
 open Rewrite
+open ComRewrite
```

Comment:
I think we could do without this open, but just a nit.

Reply:
But then we need to qualify the calls to the functions. Since we're already opening Rewrite I don't see why we shouldn't also open ComRewrite. Otherwise we should *also* remove Rewrite.

Reply:
Just a nit, it is ok.

Reply:
Actualy the `open` made my review much more difficult, I had to checkout the PR, remove the open, and compile, to see the code affected.